### PR TITLE
dotCMS/core#17321 PageAPI is not rendering containers properly

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java [hunk 1]
```diff
@@ -326,13 +328,7 @@ private Container geContainerById(final String containerIdOrPath, final User use
...
-                        // contentPrintableMap.put("personalization", personalization); // todo: not sure if this will be needed by FOX
-                        cListAsMaps.add(contentPrintableMap);
-
-                        if (personalizedContentlet != null &&
-                                personalizedContentlet.getPersonalization().equals(includeContentFor)) {
-                            personalizedContentletMap.add(contentPrintableMap);
-                        }
+                        personalizedContentletMap.add(contentPrintableMap);
                     } catch (IOException e) {
                         throw new DotStateException(e);
```
Comment (jcastro-dotcms) [inline dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java]: Shouldn't we log debugging information here?

**Review by jcastro-dotcms — CHANGES_REQUESTED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java [hunk 2]
```diff
@@ -553,4 +555,54 @@ public void testRenderPersonalizationVersion() throws DotDataException, DotSecur
...
+        when(initDataObject.getUser()).thenReturn(APILocator.systemUser());
+
+        final Collection<String> containersId = pageRenderTest.getContainersId();
+
+        for (final String id : containersId) {
+            final Container container = pageRenderTest.getContainer(id);
+            pageRenderTest.createContent(container);
+        }
+
+        Thread.sleep(500);
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java]: Why this @freddyucv

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java [hunk 3]
```diff
@@ -553,4 +555,54 @@ public void testRenderPersonalizationVersion() throws DotDataException, DotSecur
...
     }
+
+    /***
+     * Should render containers
+     *
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void testShoudrendercContainers() throws DotDataException, DotSecurityException, InterruptedException {
```
Comment (nollymar) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java]: I think the test should be renamed to `testShouldRenderContainers`

Comment (dsilvam) [reply dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java]: Actually we are missing things in the name. Please give a better name following the `methodToTest_GivenScenario_ExpectedResult` or fill a better description in the javadoc section

**Review by nollymar — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Pull request merged**